### DISCUSSION TITLE
#C++ code editor

### DISCUSSION THREAD

**Discussion by PHANTOM-XOTIC** [Sketchware-Pro/Sketchware-Pro#501] [Ideas]
A C++ code editor for opening and editing so libraries 
it is needed
Currently this is not possible

**Comment by j-px** [comment]
Sketchware Pro can't and probably never will be able to compile C/C++ code, so no editor is needed. If you really need one, use another app.

I think you don't know what .so files contain. Try editing a binary file (e.g. an image, or even better, an .so file!) with a text editor, that won't work.

**Reply by PHANTOM-XOTIC** [reply]
Ok
